src/app/records/PresentationNodeLeaf.tsx

```diff
@@ -62,3 +70,65 @@ export default function PresentationNodeLeaf({
     </>
   );
 }
+
+/**
+ * Sorts the passed node if it has a DimRecord[] | DimCollectible[] | DimMetric[]
+ */
+function sortRecords(node: DimPresentationNodeLeaf): DimPresentationNodeLeaf {
+  const sortedNode = { ...node };
+  // RECORDS
+  if (sortedNode.records) {
+    sortedNode.records = sortBy(sortedNode.records, (record) => {
+      //  Triumph is already completed so move it to back of list.
```

Comment:
https://github.com/DestinyItemManager/DIM/pull/9602#discussion_r1264278308

Don't modify the node, don't copy the node, just return a sorted list to display for each type of child list.